src/biome/text/pipeline.py

```diff
@@ -711,23 +723,45 @@ def create_vocabulary(self, config: VocabularyConfiguration) -> None:
         config
             Specifies the sources of the vocabulary and how to extract it
         """
+        # TODO: Do we have to create an empty vocab here, can't we just use the current pipeline vocab?
+        #       If we use the pipeline vocab, the call to `self._add_transformers_vocab` is not necessary!
+        vocab = self._extend_vocabulary(self._create_empty_vocab(), config)
+        self._model = self.__model_from_config(self.config, vocab=vocab)
+
+        self._add_transformers_vocab_if_necessary(vocab)
+
+    def _create_empty_vocab(self) -> Vocabulary:
+        """Creates an empty Vocabulary with configured namespaces
+
+        Returns
+        -------
+        empty_vocab
+            If necessary, the transformers namespace is added to the `non_padded_namespace`.
+        """
         # Following is a hack, because AllenNLP handles the Transformers vocab differently!
         # The transformer vocab has its own padding and oov token, so we add it to the non_padded_namespaces.
         # AllenNLP gives its "transformer vocab" by default the "tags" namespace, which is a non_padded_namespace ...
         # If we do not do this, then writing the vocab to a file and loading it will fail, since AllenNLP will
         # look for its default OVV token in the vocab unless it is flagged as non_padded_namespace.
         # (see the doc string of `allennlp.data.token_indexers.PretrainedTransformerIndexer`)
-        if TransformersFeatures.namespace in self.backbone.featurizer.indexer:
+        if self._config.features.transformers is not None:
             empty_vocab = Vocabulary(
                 non_padded_namespaces=DEFAULT_NON_PADDED_NAMESPACES
                 + (TransformersFeatures.namespace,)
             )
         else:
             empty_vocab = Vocabulary()
```

Comment:
Here, we can always create empty vocabulary with transformers namespace as non padded ns, isn't it?
```suggestion
        
        empty_vocab = Vocabulary(
            non_padded_namespaces=DEFAULT_NON_PADDED_NAMESPACES
            + (TransformersFeatures.namespace,)
        )
```